operator/src/main/java/oracle/kubernetes/operator/wlsconfig/WlsServerConfig.java

```diff
@@ -308,6 +317,10 @@ public WlsServerConfig setAdminPort(int adminPort) {
     return this;
   }
 
+  public PortDetails getAdminPortDetails() {
```

Comment:
Is this method added for future use, it doesn't seem to be used anywhere?

Reply:
same as before.